src/Update/Updates.php

```diff
@@ -627,4 +627,26 @@ public function update_9002000() {
     $this->updater->getOutput()->writeln($formattedBlock);
     $this->updater->getOutput()->writeln("");
   }
+
+    /**
+   * 9.2.1.
```

Comment:
Should this be 9.2.1 since it's being applied to 9.2.0 and not 9.2.1 (presumably)?

Reply:
I deliberately made the hook 9.2.1 to ensure that the factory hook templates were refreshed for both post-install and db-update since db-update was merged previously in #3166. In looking back through the other hooks again, it doesn't seem like this has been a standard practice so I will remove the 9.2.1 hook